src/pseudopeople/loader.py

```diff
@@ -128,3 +131,13 @@ def combine_joint_filers(data: pd.DataFrame) -> pd.DataFrame:
     joint_1040 = pd.concat([reference_persons_wide, independent_filers])
 
     return joint_1040
+
+
+def combine_ssn_and_itin_columns(data: pd.DataFrame) -> pd.DataFrame:
+    # This combines the ssn and itin columns into the ssn column.
+    # Simulants can either have an ssn or an itin so we will replace
+    # the nans in the ssn column with that rows corresponding itin value
+    data[COLUMNS.ssn.name] = np.where(
```

Comment:
This works but seems more like how we do things to do `data.loc[data[COLUMNS.ssn.name].isna(), COLUMNS.ssn.name] = data[COLUMNS.itin.name]`

I think staying in pandas is likely faster as well?

Reply:
I am running into issues doing the call in pandas because of the categorical dtype so will leave it using numpy.